src/skeptic/lib.rs

```diff
@@ -742,14 +751,14 @@ pub mod rt {
 #[test]
 fn test_omitted_lines() {
     let lines = &[
```

Comment:
it would be even better to use multi-line string literal instead of slice of str's

so we would have 

```rust
   let lines = r###"# use std::collections::BTreeMap as Map;
#
#[allow(dead_code)]
fn main() {
    let map = Map::new();
    #
    # let _ = map;
}"###;
```